tests/4.5/target_teams_distribute/test_target_teams_distribute_default_none.F90

```diff
@@ -0,0 +1,90 @@
+!===--- test_target_teams_distribute_default_none.F90-----------------------===//
+!
+! OpenMP API Version 4.5 Nov 2015
+!
+! This tests uses the default(none) clause on a target teams distribute test.
+! The test aims to validate that all values will not have default data sharing
+! attributes.
+!
+!===------------------------------------------------------------------------===//
+
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_target_teams_distribute_default_none
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+  OMPVV_TEST_OFFLOADING
+
+  CHARACTER(len=300) :: infoMessage
```

Comment:
This definition must be located before the macro of line 21. Fortran definiitions must all happen at the beginning

Reply:
Fixed on master